src/main/java/org/isf/permissions/manager/GroupPermissionManager.java

```diff
@@ -72,12 +77,31 @@ public GroupPermission create(UserGroup userGroup, Permission permission) throws
 		return operations.create(groupPermission);
 	}
 
+	@Transactional
+	public List<Permission> update(UserGroup userGroup, List<Integer> permissionIds) throws OHDataValidationException {
+
+
+		var permissions = permissionOperations.findByIdIn(permissionIds).stream().toList();
```

Comment:
To the best of my knowledge this is the only place that uses `'var`

Reply:
Sorry @dbmalkovsky ! I'm not understanding.

Reply:
This is the only place where I see `var` being used instead of the explicit data type like `List`

Reply:
@mwithi do you have an opinion about using `var`?   There are currently no other classes using it in the core repo.

Reply:
`var` should be limited to special context like local variables, for loops, and try-with-resources statements. Here is used as stream, so despite it is working I would not use it, even because I don't see the problem in just declare the correct type for `permissions`...

Reply:
I've just replaced with explicite types. Is that correct now ?